src/flow/src/adapter/flownode_impl.rs

```diff
@@ -34,11 +34,19 @@ use crate::error::InternalSnafu;
 use crate::metrics::METRIC_FLOW_TASK_COUNT;
 use crate::repr::{self, DiffRow};
 
-fn to_meta_err(err: crate::error::Error) -> common_meta::error::Error {
+/// TODO(discord9): add location
```

Comment:
What does the todo mean? Looks like the function already accepts the location parameter.